assets/src/components/button/style.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * Button styles,
+ */
+
+
+.newspack-button {
+	.components-button {
+	    padding: 0 20px 1px;
+		text-align: center;
+
+		font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif;
+		font-size: 14px;
+		line-height: 18px;
+		font-weight: 500;
+		color: $muriel-gray-700;
+		text-shadow: none;
+
+		&.is-button {
+			background-color: $muriel-white;
+			border: none;
+			border-radius: 3px;
+			cursor: pointer;
+			height: 40px;
+			max-width: 360px;
+			max-height: 40px;
+			margin-top: 13px;
+			margin-bottom: 9px;
+
+			&:hover {
+				border-color: $muriel-gray-500;
+			}
+
+			&:active:enabled {
+				border-width: 2px 1px 1px;
+			}
+		}
+
+		&.is-primary {
+			background-color: $muriel-hot-pink-500;
+			border-color: #992053;
+			box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
+			color: $muriel-white;
+
+			&:hover,
+			&:focus:enabled {
+				background-color: $muriel-hot-pink-400;
+				color: $muriel-gray-0;
+			}
+
+			&:active:enabled {
+				background-color: $muriel-hot-pink-700;
+				color: $muriel-white;
+			}
+		}
+
+		&.is-tertiary {
+			color: $muriel-hot-pink-500;
+
+			// We need this huge selector to override core styles.
+			&:not(:disabled):not([aria-disabled="true"]):not(.is-default):hover {
+				color: $muriel-hot-pink-700;
+			}
+
+			&:active:focus:enabled {
+				border: 0;
+			}
+		}
+
+	}
+}
+
+
+.newspack-card {
```

Comment:
Let's remove all of the `newspack-card` special-case styles. I don't think we need it, and even if we did we're aiming for reusable components so this would be more appropriate elsewhere. Other CSS changes I've suggested elsewhere should cover all of these styles.

Reply:
I think we need this to make sure that the button is aligned properly when within a card component, to match up with what's presented in Figma